parsing/src/processor.ts

```diff
@@ -84,11 +91,21 @@ async function processFile(inputFile: string, outputFile: string) : Promise<{ bu
         const buildings = getPowerConsumptionForBuildings(data, producingBuildings);
 
         // Pass the producing buildings with power data to getRecipes to calculate perMin and powerPerProduct
-        const recipes = getRecipes(data, buildings);
-
+        let recipes = getProductionRecipes(data, buildings);
         removeRubbishItems(items, recipes);
         fixTurbofuel(items, recipes);
 
+        //IMPORTANT: The order here matters - don't run this because fixing the turbofuel. 
+        let powerGenerationRecipes: any[] = getPowerGeneratingRecipes(data, items);
+    
+        // merge the powerGenerationRecipes with the recipes, if this feature flad is on
+        const mergePowerGenerationInRecipes: boolean = false;
```

Comment:
Typecasting to `:boolean` is not required when it's inititalized as a boolean, typescript prevents you from changing types.